modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java

```diff
@@ -50,6 +53,7 @@
 	public ObjectDefinitionsDetailsDisplayContext(
 		HttpServletRequest httpServletRequest,
 		ObjectDefinitionLocalService objectDefinitionLocalService,
+		ObjectEntryManagerRegistry objectEntryManagerRegistry,
 		ModelResourcePermission<ObjectDefinition>
 			objectDefinitionModelResourcePermission,
```

Comment:
@murilostodolni I believe you should place the `objectEntryManagerRegistry` after the `objectDefinitionModelResourcePermission`